common/source/docs/common-choosing-a-ground-station.rst

```diff
@@ -1,5 +1,5 @@
 .. _common-choosing-a-ground-station:
-[copywiki destination="plane,copter,rover,blimp"]
+[copywiki destination="plane,copter,rover,blimp,sub"]
```

Comment:
I'm reluctant to add this page to Sub without some site-specific recommendations (or filtering) based on options that have actually been tested and confirmed to work with ArduSub vehicles.

There have been quite a few modifications made to QGroundControl over the years to make it properly compatible, and I expect several of the other control station software options will expect a radio connection (which Sub doesn't support), and may not have support for things like Sub's joystick button functions via `MANUAL_CONTROL` messages.

I can add an initial version of that in a separate PR next week, which this one can get rebased over, if that works for you?

Reply:
we can restrict on only QGC for sub easily